bin/update_upgrade_containers.sh

```diff
@@ -0,0 +1,44 @@
+import os
```

Comment:
Add a #!/usr/bin/env python3 at the beginning of the file:

```suggestion
#!/usr/bin/env python

import os
```